bedrock/newsletter/templates/newsletter/management.html

```diff
@@ -72,6 +72,8 @@ <h1>{{ ftl('newsletters-manage-your-email-preferences') }}</h1>
       data-recovery-url="{{ recovery_url }}"
       data-unsubscribe-url="{{ unsubscribe_url }}">
 
+      <input type="hidden" name="source_url" value="{{ request.build_absolute_uri(request.path) }}">
```

Comment:
The first time I load the page (before the token is removed from the path), I can still see the token being inserted here?

<img width="865" alt="image" src="https://github.com/mozilla/bedrock/assets/400117/f2ceef1a-2a8b-4cb5-ae30-44ef5ddeda7c">

Reply:
Oh, right. It's not a query parameter, it's part of the URL. I'll come up with a fix.

Reply:
Moved the logic to the view to create a URL with no token. It should be good now.